Test/TestProject/TestProject.csproj

```diff
@@ -21,7 +21,7 @@
     <Import Project="../../Source/dafny.msbuild/build/dafny.msbuild.props" />
     <Import Project="../../Source/dafny.msbuild/build/dafny.msbuild.targets" />
 
-    <PropertyGroup>
-        <DafnyVerificationTimeLimit>100</DafnyVerificationTimeLimit>
-    </PropertyGroup>
+    <ItemGroup>
+        <VerifyDafnyPassthrough Include="timeLimit:100" />
```

Comment:
I know it's a pain to test, but did you verify this overrides the value in dafny.msbuild.props?

Reply:
Yup! It does. See comment above: https://github.com/dafny-lang/dafny.msbuild/pull/11#discussion_r403261233